### DISCUSSION TITLE
How can one disable/hide the menu bar?

### DISCUSSION THREAD

**Discussion by tello2004** [orbitalquark/textadept#381] [Q&A] [answered]
I tried with `textadept.menu.menubar = nil` (as seen in [Manual](https://orbitalquark.github.io/textadept/manual.html#running), at the **Running** section), but it still shows up, both in latest stable and latest nightly.

Is there a way to actually get rid of it?

**Comment by jxzwp** [comment, marked as answer]
Put the following in init.lua
```lua
events.connect(events.INITIALIZED, function()
  textadept.menu.menubar = nil
end)
```

You can only turn off the menu after Textadept has finished initializing.
